definitions/divorce/json/CaseEventToFields/CaseEventToFields.json

```diff
@@ -9858,341 +9858,6 @@
-    "PageDisplayOrder": 1,
-    "PageColumnNumber": 1,
-    "ShowSummaryChangeOption": "Yes"
-  },
-  {
-    "LiveFrom": "20/09/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseEventID": "EnterAOSOffline_Respondent",
-    "CaseFieldID": "D8RespondentHomeAddress",
-    "PageFieldDisplayOrder": 15,
-    "DisplayContext": "OPTIONAL",
-    "PageID": "aosdataentry",
-    "PageLabel": "Enter scanned form case data",
-    "PageDisplayOrder": 1,
-    "PageColumnNumber": 1,
-    "ShowSummaryChangeOption": "Yes"
-  },
-  {
-    "LiveFrom": "20/09/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseEventID": "EnterAOSOffline_Respondent",
-    "CaseFieldID": "RespEmailAddress",
-    "PageFieldDisplayOrder": 17,
-    "DisplayContext": "OPTIONAL",
-    "PageID": "aosdataentry",
-    "PageLabel": "Enter scanned form case data",
-    "PageDisplayOrder": 1,
-    "PageColumnNumber": 1,
-    "ShowSummaryChangeOption": "Yes"
-  },
-  {
-    "LiveFrom": "20/09/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseEventID": "EnterAOSOffline_Respondent",
-    "CaseFieldID": "solAOSSOT",
-    "PageFieldDisplayOrder": 18,
-    "DisplayContext": "MANDATORY",
-    "PageID": "aosdataentry",
-    "PageLabel": "Enter scanned form case data",
-    "PageDisplayOrder": 1,
-    "PageColumnNumber": 1,
-    "ShowSummaryChangeOption": "Yes"
-  },
-  {
-    "LiveFrom": "20/09/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseEventID": "EnterAOSOffline_Respondent",
-    "CaseFieldID": "D8RespondentCorrespondenceAddress",
-    "PageFieldDisplayOrder": 19,
-    "DisplayContext": "OPTIONAL",
-    "PageID": "aosdataentry",
-    "PageLabel": "Enter scanned form case data",
-    "PageDisplayOrder": 1,
-    "PageColumnNumber": 1,
-    "ShowSummaryChangeOption": "Yes"
-  },
-  {
-    "LiveFrom": "20/09/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseEventID": "EnterAOSOffline_Respondent",
-    "CaseFieldID": "D8RespondentSolicitorName",
-    "PageFieldDisplayOrder": 20,
-    "DisplayContext": "OPTIONAL",
-    "PageID": "aosdataentry",
-    "PageLabel": "Enter scanned form case data",
-    "PageDisplayOrder": 1,
-    "PageColumnNumber": 1,
-    "ShowSummaryChangeOption": "Yes"
-  },
-  {
-    "LiveFrom": "20/09/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseEventID": "EnterAOSOffline_Respondent",
-    "CaseFieldID": "D8RespondentSolicitorEmail",
-    "PageFieldDisplayOrder": 21,
-    "DisplayContext": "OPTIONAL",
-    "PageID": "aosdataentry",
-    "PageLabel": "Enter scanned form case data",
-    "PageDisplayOrder": 1,
-    "PageColumnNumber": 1,
-    "ShowSummaryChangeOption": "Yes"
-  },
-  {
-    "LiveFrom": "20/09/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseEventID": "EnterAOSOffline_Respondent",
-    "CaseFieldID": "D8RespondentSolicitorPhone",
-    "PageFieldDisplayOrder": 22,
-    "DisplayContext": "OPTIONAL",
-    "PageID": "aosdataentry",
-    "PageLabel": "Enter scanned form case data",
-    "PageDisplayOrder": 1,
-    "PageColumnNumber": 1,
-    "ShowSummaryChangeOption": "Yes"
-  },
```

Comment:
Extracted and added to the -prod.json file as is. Needed to prevent duplicate errors at the same time allow these to be present as they currently are.

Reply:
File no longer required